website/showcase.json

```diff
@@ -510,5 +510,13 @@
     "infoLink": "http://x.co/FlareNews",
     "infoTitle": "Social network that connects entrepreneurs to fellow entrepreneurs, consumers, investors and experts",
     "pinned": false
+  },
+  {
+    "name": "JadoPado",
+    "icon": "https://lh3.googleusercontent.com/VX8CXya2KmFc-n9aPChiUpzIg2IkuzudWy76OmFKNMu2eQDFq9xFdDMR-U1E7Sa88eUs=w300-rw",
+    "linkAppStore": "https://itunes.apple.com/ao/app/jadopado/id1149279535",
+    "linkPlayStore": "https://play.google.com/store/apps/details?id=com.jadopado.buyer",
+    "infoLink": "https://www.entrepreneur.com/article/287838",
```

Comment:
Do you have a technical article describing your use of React Native in this app?